basics/devtools.md

```diff
@@ -0,0 +1,145 @@
+# Inspecting the Vivaldi UI with DevTools
+
+Author(s): luetage
+
+Developer Tools allow us to inspect Vivaldi’s user interface and debug our
+modifications. Trying to access DevTools the usual way (keyboard shortcuts,
+inspect from menu, etc.) won’t necessarily give us access to the interface,
+however. In the following section we want to look at different ways how to
+achieve this.
+
+## Loading DevTools
+
+Loading DevTools for the UI requires us to open one out of a set of specific
+pages first.
+
+- Open **`vivaldi://inspect/#apps/`** and click “inspect” beneath the first line
+  (Vivaldi browser.html).
+- Open **`vivaldi://settings/`** (in a **tab**), or **`vivaldi://experiments/`**,
+  or **`vivaldi://mail/`** and load DevTools with a keyboard shortcut, Quick
+  Commands, or menu entry (e.g. `Main Menu → Tools → Developer Tools`).
+- Enable “Show Introduction” for private windows in `vivaldi://settings/privacy/`.
```

Comment:
Not necessary I think.
Also, in this list, maybe you could recommend a certain method.

Reply:
Thank you for sharing your opinion. This guide collects all methods. Whatever works for you is the recommended method.